remixer_ui/src/main/java/com/google/android/libraries/remixer/ui/view/RemixerFragment.java

```diff
@@ -79,6 +84,35 @@ public void onClick(View view) {
   }
 
   /**
+  /**
+   * Attach this instance to a shake gesture using {@code sensorManager } and show fragment when magnitude exceeds {@code threshold}
+   */
+  public void attachToShake(final FragmentActivity activity, SensorManager sensorManager, final double threshold) {
+    this.sensorEventListener = new SensorEventListener() {
+      double lastMagnitude;
+      /* Temporary delay until `show` bug is fixed */
+      long lastSpike = 0;
+
+      @Override
+      public void onSensorChanged(SensorEvent event) {
+        double currentMagnitude = Math.sqrt(event.values[0] * event.values[0] + event.values[1] * event.values[1] + event.values[2] * event.values[2]);
```

Comment:
Fixed! I should brush up on the Google Java style guide. My brain is in unregulated javascript mode.